components/suggest/src/store.rs

```diff
@@ -1664,6 +1706,64 @@ mod tests {
                 ]
                 "#]],
             ),
+            (
+                "keyword = `soft`; non-sponsored only",
```

Comment:
Let's also add a test for `soft li`, so that we can make sure the prefix matching works.